storage/storage-samples/storage-rv-sample/src/main/java/org/fido/iot/storage/To0DbStorage.java

```diff
@@ -98,6 +100,100 @@ public long storeRedirectBlob(Composite voucher, long requestedWait, byte[] sign
     throw new ResourceNotFoundException(guid.toString());
   }
 
+  @Override
+  public List<String> getOvKeysAllowlist() {
+
+    List<String> keysInAllowList = new ArrayList<>();
+
+    String sql = "SELECT PUBLIC_KEY_HASH FROM OV_KEYS_ALLOWLIST";
+
+    try (Connection conn = dataSource.getConnection();
+        PreparedStatement pstmt = conn.prepareStatement(sql)) {
+
+      try (ResultSet rs = pstmt.executeQuery()) {
+        while (rs.next()) {
+          keysInAllowList.add(rs.getString(1));
+        }
+      }
+
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
+    }
+    return keysInAllowList;
+  }
+
+  @Override
+  public void setOvKeysAllowlist() {
+
+    List<String> allowlist = new ArrayList<>();
+    allowlist.add("42110E8F0F3184A1A5C51868BCBFF7144D66E41D1A188103C0264D5DA8BBCF88");
```

Comment:
Would be better to read these hashes from configuration, rather than these being hard-coded here

Reply:
Done